File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL0001U00GrdSlipInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
    public class CPL0001U00GrdSlipInfo
    {
        private String _slipCode;
        private String _slipName;
        private String _slipNameRe;
        private String _jundalGubun;
        private String _rowState;

        public String SlipCode
        {
            get { return this._slipCode; }
            set { this._slipCode = value; }
        }

        public String SlipName
        {
            get { return this._slipName; }
            set { this._slipName = value; }
        }

        public String SlipNameRe
        {
            get { return this._slipNameRe; }
            set { this._slipNameRe = value; }
        }

        public String JundalGubun
        {
            get { return this._jundalGubun; }
            set { this._jundalGubun = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public CPL0001U00GrdSlipInfo() { }

        public CPL0001U00GrdSlipInfo(String slipCode, String slipName, String slipNameRe, String jundalGubun, String rowState)
        {
            this._slipCode = slipCode;
            this._slipName = slipName;
            this._slipNameRe = slipNameRe;
            this._jundalGubun = jundalGubun;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL0001U00GrdSlipInfo.java
package nta.med.data.model.ihis.cpls;

public class CPL0001U00GrdSlipInfo {
	private String slipCode;
	private String slipName;
	private String slipNameRe;
	private String jundalGubun;

	public CPL0001U00GrdSlipInfo(String slipCode, String slipName,
			String slipNameRe, String jundalGubun) {
		super();
		this.slipCode = slipCode;
		this.slipName = slipName;
		this.slipNameRe = slipNameRe;
		this.jundalGubun = jundalGubun;
	}

	public String getSlipCode() {
		return slipCode;
	}

	public void setSlipCode(String slipCode) {
		this.slipCode = slipCode;
	}

	public String getSlipName() {
		return slipName;
	}

	public void setSlipName(String slipName) {
		this.slipName = slipName;
	}

	public String getSlipNameRe() {
		return slipNameRe;
	}

	public void setSlipNameRe(String slipNameRe) {
		this.slipNameRe = slipNameRe;
	}

	public String getJundalGubun() {
		return jundalGubun;
	}

	public void setJundalGubun(String jundalGubun) {
		this.jundalGubun = jundalGubun;
	}

}
